yt/yt/server/master/cell_master/program.h

```diff
@@ -201,13 +213,27 @@ class TCellMasterProgram
         _exit(0);
     }
 
+private:
+    void PrintCompatibilityInfoAndExit() const
+    {
+        NYson::TYsonWriter writer(&Cout, NYson::EYsonFormat::Pretty);
+        auto node = NYTree::BuildYsonNodeFluently()
+          .BeginMap()
+              .Item("current_reign").Value(static_cast<int>(NCellMaster::GetCurrentReign()))
```

Comment:
ToUnderlying looks better

Reply:
actually it was TReign already, removed unnecessary cast